UndertaleModTool/Editors/UndertaleEmbeddedTextureEditor.xaml.cs

```diff
@@ -69,6 +69,9 @@ public UndertaleEmbeddedTextureEditor()
 
         private void UpdateImage(UndertaleEmbeddedTexture texture)
         {
+            if (texture.TextureData?.Image is null)
```

Comment:
If the `Image` property itself is set to null, we may want to actually clear the image sources in the UI, so they disappear.

Reply:
I think I added what you were after. Is that the way you wanted it done?

Reply:
Looks good to me!